### ISSUE TITLE
[BUG] ServiceWorker tries to precache assets from /static-files/static-files instead of /static-files

### ISSUE THREAD

**Issue by jpambrun** [mlflow/mlflow#1799]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in MLflow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux
- **MLflow installed from (source or binary)**:  using `pip install mlflow`
- **MLflow version (run ``mlflow --version``)**: 1.2.0
- **Python version**: 3.6.6
- **npm version, if running the dev UI**: N/A
- **Exact command to reproduce**: `mlflow server --backend-store-uri sqlite:////tmp/mlflow.db --default-artifact-root file:////tmp/ --host 0.0.0.0`

### Describe the problem
The serviceworker is issuing request to `/static-files/static-files/somefile.css` instead of `/static-files/somefile.css`. This cause the initial load to be very slow especially with the default 4 gunicorn workers. 

For instance, requests to `http://localhost:5000/asdf/static-files/static-files/static/css/main.708e4447.css` fails while http://localhost:5000/asdf/static-files/static/css/main.708e4447.css` would work.

![image](https://user-images.githubusercontent.com/718384/64030091-e21ed780-cb13-11e9-8f38-7ab596250547.png)

**Comment by ustcscgyer** [comment]
I can take a look at this issue.

**Comment by ustcscgyer** [comment]
Hi @jpambrun, it seems that I cannot reproduce the error:

![image](https://user-images.githubusercontent.com/12961341/65000867-26470180-d8a2-11e9-953d-648ccd9f2008.png)

**Comment by jpambrun** [comment]
Still happens here. I still have the `static-files/static-files/static/` prefix for every request originating from the service worker.

![image](https://user-images.githubusercontent.com/718384/65040984-b5e0c480-d923-11e9-93fe-58b98d05b3b4.png)

I am not sure how I could have broke this from the config/user side. I run it in kubernetes (see config below), then I use kubectl proxy to reach it. Everything is fine expect the urls used by the service worker. What's most is that it stalls all other requests for several seconds (with the default configuration, adding `-w 24` provide some relief as show in the screenshot) while these requests are pending. 

``` dockerfile
FROM python:3.6
RUN pip install mlflow==1.2.0 google-cloud-storage sqlalchemy psycopg2-binary
RUN mkdir -p /mlflow/
WORKDIR /mlflow/
EXPOSE 5000

ENV BACKEND_URI some_postgres_url
ENV ARTIFACT_ROOT some_gcs_bucket

CMD pwd && mlflow server -w 24 \
     --backend-store-uri ${BACKEND_URI} \
     --default-artifact-root ${ARTIFACT_ROOT} \
     --host 0.0.0.0
```
``` yaml
kind: Namespace
apiVersion: v1
metadata:
  name: mlflow
---
apiVersion: apps/v1
kind: Deployment
metadata:
  namespace: mlflow
  name: mlflow-deploy
spec:
  strategy:
    type: Recreate
  selector:
    matchLabels:
      app: mlflow
  replicas: 1
  template:
    metadata:
      labels:
        app: mlflow
    spec:
      containers:
        - name: mlflow
          image: image_from_provided_dockerfile
          imagePullPolicy: Always
          ports:
            - containerPort: 5000
---
apiVersion: v1
kind: Service
metadata:
  name: mlflow-svc
  namespace: mlflow
  labels:
    app: mlflow
spec:
  ports:
    - name: http
      port: 80
      protocol: TCP
      targetPort: 5000
  selector:
    app: mlflow
```

**Comment by junchen1992** [comment]
Same issue here. Have you got a fix now? @jpambrun

**Comment by jpambrun** [comment]
I am still on 1.2 so I don't know if it affects 1.3. I don't know of a fix or workaround. Starting mlflow with more workers (w.g. `mlflow server -w 24`) makes it bearable. Otherwise the few workers are stalled for a long very time (300-400ms per file) replying with 404s.

**Comment by jpambrun** [comment]
I just upgraded to 1.4. The issues is still there.

**Comment by jpambrun** [comment]
The stale bot is a bit aggressive. It feels quite a bit spammy from my perspective.

**Comment by jpambrun** [comment]
Bump..

**Comment by jpambrun** [comment]
I am not sure if it's fixed in 1.5, but this bot is fucking annoying.

**Comment by jpambrun** [comment]
I could no reproduce in 1.7.1. Closing.
